indra/llwindow/llgamecontrol.cpp

```diff
@@ -309,11 +446,17 @@ namespace
 
     constexpr U8 MAX_AXIS = NUM_AXES - 1;
     constexpr U8 MAX_BUTTON = NUM_BUTTONS - 1;
+    constexpr U8 MAX_AXIS_DEAD_ZONE = 255;
+    constexpr U8 MAX_AXIS_OFFSET = 255;
```

Comment:
The raw axis value is a `S16`, so `MAX_AXIS_DEAD_ZONE` should be `S16` and probably higher than `255`... maybe even as high as 2^14 = `16384`.  Similarly, `MAX_AXIS_OFFSET` should be `S16`.

Reply:
I gave these values randomly, hoping that you provide the correct values :-)

Reply:
Sure, DEAD_ZONE can be explicitly positive but applied symmetrically on the negative side, however, since we'll be doing **math** with S16 axis measurement the DEAD_ZONE should also be S16.

An axis measurement will range positive and negative and that channel may very well need a negative OFFSET adjustment.

Reply:
The value of MAX_AXIS_DEAD_ZONE is also always positive
It doesn't make sense to set a negative limit for mDeadZone

Moreover, I would like to be sure that mDeadZone is positive
That's why I use the unsigned integer type (U8 replaced with U16)